### ISSUE TITLE
HTSeq.GenomicInterval_from_directional AttributeError: 'str' object has no attribute 'se'

### ISSUE THREAD

**Issue by justinaruda** [htseq/htseq#62]
Hi there,

I'm running into `AttributeError: 'str' object has no attribute 'se'`  trying to use the from_directional method. Looking at the source in src/HTSeq/_HTSeq.pyx it seems this may be due to a typo in how the strand string is handled (.se is called erroneously).

The following code generates the error on version 2.0.2:

```
HTSeq.GenomicInterval_from_directional("chr5", 5, 125, strand="-")
```

**Comment by iosonofabio** [comment]
Thank you @justinaruda - I'll try to take a look this afternoon

**Comment by iosonofabio** [comment]
Fixed in b6c55bc and added test. This will be released in 2.0.3, hopefully today.

Thanks again for reporting.
